derive/src/decode.rs

```diff
@@ -83,17 +83,42 @@ fn create_decode_expr(field: &Field, input: &TokenStream) -> TokenStream {
 		).to_compile_error();
 	}
 
+	#[cfg(feature = "std")]
+	let err_msg = format!("Error decoding field {}", name);
```

Comment:
You don't need to distinguish here.

Reply:
I am afraid that if we don't explicitly trim these strings, they will not be elimintad from the resulting code by DCE